### DISCUSSION TITLE
[Win11] Refresh window in the file explorer context menu

### DISCUSSION THREAD

**Discussion by sluggogle** [ramensoftware/windhawk-mods#71] [Mod ideas]
Since Windows 11, the new file explorer context menu is missing the refresh option. You have to "Show more options" and display the old context menu to choose it.

Would be great to see it directly in the new context menu. There is no ressource on the internet to make it possible changing something in the registry.

**Comment by Anixx** [comment]
You can make it use the old menu directly, without the Win11 menu. Thre are plenty of methods on the internet.
